UndertaleModTool/Editors/UndertaleCodeEditor.xaml.cs

```diff
@@ -372,6 +372,12 @@ private async void DecompileCode(UndertaleCode code)
                                                 possibleObjects.Add(data.Fonts[id]);
                                             if (id < data.Sounds.Count)
                                                 possibleObjects.Add(data.Sounds[id]);
+                                            if (id < data.Shaders.Count)
+                                                possibleObjects.Add(data.Shaders[id]);
+                                            if (id < data.Extensions.Count)
```

Comment:
Can extensions actually be referenced in scripts by ID? If not, then this is just cluttering up the context menu list.

Reply:
I kind of recall that being possible somewhere, but now I'm not so sure. I put it in just to be safe, but if it's unnecessary, I guess it's okay to get rid of it.